cacheops/query.py

```diff
@@ -524,18 +526,19 @@ def invalidate_m2m(sender=None, instance=None, model=None, action=None, pk_set=N
         instance_column, model_column = model_column, instance_column
 
     # TODO: optimize several invalidate_objs/dicts at once
+    using = kwargs['using']
```

Comment:
I meant unpacking `using` in func signature.

Reply:
Ok, it's better.